packages/editor/src/components/rich-text/index.js

```
@@ -215,17 +212,29 @@ export class RichText extends Component {
 	 * @param {PasteEvent} event The paste event as triggered by TinyMCE.
 	 */
 	onPaste( event ) {
-		const dataTransfer =
-			event.clipboardData ||
-			event.dataTransfer ||
-			this.editor.getDoc().dataTransfer ||
-			// Removes the need for further `dataTransfer` checks.
-			{ getData: () => '' };
-
-		const { items = [], files = [], types = [] } = dataTransfer;
+		const clipboardData = event.clipboardData;
+		const { items = [], files = [] } = clipboardData;
 		const item = find( [ ...items, ...files ], ( { type } ) => /^image\/(?:jpe?g|png|gif)$/.test( type ) );
-		const plainText = dataTransfer.getData( 'text/plain' );
-		const HTML = dataTransfer.getData( 'text/html' );
+		let plainText = '';
+		let HTML = '';
+
+		try {
+			plainText = clipboardData.getData( 'text/plain' );
+			HTML = clipboardData.getData( 'text/html' );
+		// IE11 will throw an invalid argument error.
+		} catch ( e1 ) {
+			try {
+				HTML = clipboardData.getData( 'Text' );
+			} catch ( e2 ) {
+				return;
```

Do we want to silently fail here? Isn't this a legit error?

---

returning here will not prevent default behaviour, so just don't do anything special here, not fail silently. I'll leave a comment.

---

Oh, right, sorry! 😓